en/sql/function/json_fn.rst

```diff
@@ -1128,3 +1151,203 @@ The json path arguments are evaluated one by one, from left to right. The result
       json_remove('{"a":1,"b":2}','$.a')
     ======================
       {"b":2}
+
+.. _fn-json-table:
+
+JSON_TABLE
+=====================
+
+**JSON_TABLE** facilitates transforming jsons into a table-like structures
+that can be queried similarly as regular tables.
+The transformation generates a single row or multiple rows, by expanding for
+example the elements of a JSON_ARRAY.
+
+The full syntax of **JSON_TABLE**:
+::
+
+    JSON_TABLE(
+        expr,
+        path COLUMNS (column_list)
+    )   [AS] alias
+
+
+    <column_list>::=
+        <column> [, <column>] ...
+
+    <column>::=
+        name FOR ORDINALITY
+	|  name type PATH string path <on_empty> <on_error>
+	|  name type EXISTS PATH string path
+	|  NESTED [PATH] path COLUMNS <column_list>
+
+    <on_empty>::=
+        NULL | ERROR | DEFAULT value ON EMPTY
+
+    <on_error>::=
+        NULL | ERROR | DEFAULT value ON ERROR
+
```

Comment:
I think we need to add some detail about what are name, path/string_path, value, expr, alias